sky/provision/runpod/instance.py

```diff
@@ -83,7 +83,9 @@ def run_instances(region: str, cluster_name_on_cloud: str,
                 name=f'{cluster_name_on_cloud}-{node_type}',
                 instance_type=config.node_config['InstanceType'],
                 region=region,
-                disk_size=config.node_config['DiskSize'])
+                disk_size=config.node_config['DiskSize'],
+                image_name=config.node_config['ImageId'] or 'runpod/base:0.0.2',
```

Comment:
With the handling of `image_id` in `runpod.py`, we do not need the `or` logic here.
```suggestion
                image_name=config.node_config['ImageId'],
```